src/components/FilterBar.jsx

```diff
@@ -82,18 +65,16 @@ export default function FilterBar(props) {
         <FilterDropDown
           label="Sorting"
           items={sortingOptions}
-          defaultValue="deadlineSoon"
+          value="sorting"
```

Comment:
Why is this fixed? The value is just whatever key of `sortingOptions` should be currently selected, e.g. `deadlineAsc`. You can determine the value from the `queryParams` object that was passed in.